datasource/src/main/scala/quasar/physical/mongo/MongoConfig.scala

```diff
@@ -20,18 +20,18 @@ import slamdata.Predef._
 
 import argonaut._, Argonaut._
 
-final case class MongoConfig(connectionString: String)
+final case class MongoConfig(connectionString: String, bsonBatchSize: Option[Int])
```

Comment:
How about naming this field something like `resultBatchSizeBytes` for a bit more self-documentation?